### ISSUE TITLE
Page - Create certificate

### ISSUE THREAD

**Issue by raysummee** [Megabyte-143/certified_cliche#23]

**Comment by Abhinav0909** [comment]
Hey,I want to work on this issue

**Comment by Abhinav0909** [comment]
Ok I want to work on this issue first

**Comment by raysummee** [comment]
@Abhinav0909 any update from your side. Please stick to the time limit. For any doubt or any help you can ping on the discord channel.

**Comment by Abhinav0909** [comment]
Yeah sorry for the delay.I will submit by day after tomorrow.sorry for delay

**Comment by raysummee** [comment]
@Abhinav0909 Sure. Take your time. Let me know if you have any doubt or any question.

**Comment by Megabyte-143** [comment]
Hey @Abhinav0909, I hope you have completed the issue. If not, then please complete it by today cause it's a Level2, else we have to assign it to someone else.

**Comment by Abhinav0909** [comment]
Ok

**Comment by Abhinav0909** [comment]
Hey,so I have to make a dropdown right in select the event field

**Comment by Abhinav0909** [comment]
right

**Comment by Abhinav0909** [comment]
![Screenshot (404)](https://user-images.githubusercontent.com/71188751/157055859-82e8f8cb-0647-48c7-bb6c-44c2e846342c.png)

**Comment by Abhinav0909** [comment]
I have implemented the checkbox thing but please tell me what shoud be do with chose file functionality and (select an event) dropdown thing

**Comment by raysummee** [comment]
hey, @Abhinav0909, I guess in design we don't have the choose file input. Instead of that, the user can click on the edit button on the image at the top right side and add the images. So what you have to do is that:

* hide the choose file input (use display: none;)
* Add the edit button style on the images with a gradient

You don't have to do any functionality just do the styling for now. The functionality will be done later.

PS: we need the file input for the actual upload of images, but will be hidden for the user. 

If you have any doubt please feel free to ping in the discord channel.

**Comment by Abhinav0909** [comment]
what should be the format of branch name??
I think it should be  ak/feat/ui

**Comment by raysummee** [comment]
@Abhinav0909 You can create branch of any name. It's up to you. `ak/feat/ui` looks good to me. What I suggest is: `feat/create-certificate`. 

Just make sure you pull to dev branch only.

**Comment by Abhinav0909** [comment]
Check it https://github.com/Megabyte-143/certified_cliche/pull/63

**Comment by Abhinav0909** [comment]
Any idea when this pr will be merged??

**Comment by raysummee** [comment]
@Abhinav0909 let me check your code. I'll merge it if no more modification needed. Thanks

**Comment by Abhinav0909** [comment]
I think I have resolved all those issues that you have mentioned

**Comment by raysummee** [comment]
@Abhinav0909 Please make sure you pull to `dev` not `main`

**Comment by Abhinav0909** [comment]
I have done that if not can you come once in discord to show me

**Comment by Abhinav0909** [comment]
Because I think I have did it

**Comment by Abhinav0909** [comment]
I have forked your repo and clone dev branch from it ,worked on it and created a new branch from it

**Comment by raysummee** [comment]
@Abhinav0909 I have given the instructions on the PR. Please see that one. Thanks

Check this comment https://github.com/Megabyte-143/certified_cliche/pull/65#issuecomment-1062869643

**Comment by Abhinav0909** [comment]
can we have a meet at discord.I have some doubts that I have to ask

**Comment by Abhinav0909** [comment]
tommorow
